README.md

```diff
@@ -1,15 +1,20 @@
 [![Build Status](https://travis-ci.org/anhnamtran/tellina_user_experiment.svg?branch=master)](https://travis-ci.org/anhnamtran/tellina_user_experiment)
 # Tellina User Experiment
+Tellina is a natural language -> command translation tool. Tellina accepts a natural language description of file system operations, and displays a ranked list of bash one-liner suggestions made by the model. The user can scroll down the web page to explore more suggestions.
```

Comment:
nit: "... file system operations ~,~ and displays ..." (remove the comma before the "and"